### PR TITLE
added page and logo for Warmup thermostat

### PR DESCRIPTION
**Pull request by alex-0103**
**Description:**
new documentation page for warmup4IE thermostat, see PR link below.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21144

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/climate.warmup.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: "Warmup Thermostat"
+description: "Instructions on how to integrate Warmup (4IE) thermostats within Home Assistant."
+date: 2019-02-27 12:01
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: warmup.png
+ha_category: Climate
+ha_release: 0.93
```
Comment (klaasnicolaas) [inline source/_components/climate.warmup.markdown]:
```suggestion
ha_release: 0.94
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/climate.warmup.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+date: 2019-02-27 12:01
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: warmup.png
+ha_category: Climate
+ha_release: 0.94
+ha_iot_class: Cloud Polling
+ha_qa_scale:
```
Comment (frenck) [inline source/_components/climate.warmup.markdown:14]: If there is no scale, this should be removed.

File: source/_components/climate.warmup.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+The `warmup` climate platform let you control Warmup 4IE thermostats from Home Assistant.
```
Comment (frenck) [inline source/_components/climate.warmup.markdown:17]:
```suggestion
The `warmup` climate platform lets you control Warmup 4IE thermostats from Home Assistant.
```

File: source/_components/climate.warmup.markdown [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+climate:
+  - platform: warmup
+    username: YOUR_E_MAIL_ADDRESS
+    password: YOUR_PASSWORD
+    location: YOUR_LOCATION_NAME
+    room: YOUR_ROOM_NAME
+```
+
+<p class='note'>
+To use this component, you have to setup an account at the warmup web site first and register your thermostat appropriately. See <https://my.warmup.com/login>.
```
Comment (frenck) [inline source/_components/climate.warmup.markdown:31]:
```suggestion
To use this component, you have to set up an account at the warmup web site first and register your thermostat appropriately. See <https://my.warmup.com/login>.
```

File: source/_components/climate.warmup.markdown [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+</p>
+
+{% configuration %}
+name:
+  description: Name of thermostat.
```
Comment (frenck) [inline source/_components/climate.warmup.markdown:36]:
```suggestion
  description: Name of the thermostat.
```

File: source/_components/climate.warmup.markdown [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  type: string
+location:
+  description: The name of the location from your Warmup account.
+  required: true
+  type: string
+room:
+  description: The name of the room from your Warmup account.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (frenck) [inline source/_components/climate.warmup.markdown:56]:
```suggestion
{% endconfiguration %}

```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @alex-0103. I've reviewed your PR and left some tiny comments. Could you please take a look? Thanks! 👍

**Pull request closed without merging**